Update PDBLoader.js with Comments

kkruups opened the pull request:
Added comments to better understand file parser and data structures in more detail.

kkruups committed:
Update PDBLoader.js with Comments

Added comments to better understand file parser and data structures in more detail.

kkruups commented:
@mrdoob per your request. This is a separate Pull Request for Comments added to file.

BR/Kk

mrdoob commented:
Thanks! Kind of agree with @Mugen87 [comment](https://github.com/mrdoob/three.js/pull/12122#issuecomment-326903866) though...

kkruups commented:
@mrdoob , @Mugen87 ,

Ok guys, understood, will update docs, as soon as I get a chance.

BR./KK

kkruups committed:
Fixed Bug in Double Byte Code in utf8Read

Code before fix was extracting 4-bits from 1st utf8 Byte instead of 5 bits, resulting in 10-bit binary number instead of 11-bit as specified for Double Byte.
